src/test/setupTests.ts

```diff
@@ -1,9 +1,16 @@
 import '@testing-library/jest-dom';
+import { insertDefaultData } from 'mocks/insertDefaultData';
 import { mockServer } from 'mocks/server';
+import { obj } from './utils';
 
 beforeAll(() => {
+    insertDefaultData(mockServer);
     mockServer.listen({
-        onUnhandledRequest: 'error'
+        onUnhandledRequest: (req) => {
+            const message = `Unexpected request: ${obj(req)}`;
+            console.error(message);
```

Comment:
is this `console.error` statement expected?

Reply:
Oh I guess not. The thrown error will end up in the test output. I'll remove it.